readme.md

```diff
@@ -124,5 +158,5 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
-
+Every member of the team can fork from the central repository, and then clone, make changes, add/commit/push/pull request back to the main repo. This way gives the owner of the central repo a bit more control about what is going on there. In project 3, my team decided to all be collaborators within one repo (another option), and the benefits were that we didn't have to constantly be making pull requests. The drawbacks were that sometimes versioning wasn't as clear.
```

Comment:
:+1: 

The instructional team uses a similar model.  We default to using PRs, for a double check of the changes and knowledge transfer.  We bypass the PR for trivial commits.